tests/ed25519/eddsa.py

```diff
@@ -1,7 +1,14 @@
 from pure25519.eddsa import H, Hint
-from pure25519.basic import (bytes_to_clamped_scalar, bytes_to_element, bytes_to_scalar, scalar_to_bytes, Base, L)
+from pure25519.basic import bytes_to_clamped_scalar, bytes_to_element, bytes_to_scalar, scalar_to_bytes, Base, L
 from pure25519.eddsa import create_signing_key, create_verifying_key
 
+"""
+This file, using pure25519 package, provides high security,
+generating key pers and signatures for the miners.
```

Comment:
```suggestion
generating key pairs and signatures for the miners.
```

Reply:
fixed in 88127f439bd8e20b30ac26cb0f739a8059935186